templates/caddy.conf

```diff
@@ -27,5 +28,5 @@ limit nofile 1048576 1048576
 script
         cd /etc/caddy
```

Comment:
> shouldn't that be changed to {{ caddy_conf_dir }}?

Apparently upstream thinks the opposite - as I received no comment on https://github.com/mholt/caddy/pull/1541/files#diff-4e618426bf81a9020795e43307fcb569R22